platforms/software/software-diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/internal/artifact/transforms/renderer/ConsoleArtifactTransformReportRenderer.java

```diff
@@ -110,7 +110,7 @@ private void writeAttribute(Integer max, String name, Object value) {
     private void writeSuggestions(List<ReportArtifactTransform> transforms) {
         output.style(StyledTextOutput.Style.Info);
         if (transforms.stream().anyMatch(t -> !t.isCacheable())) {
-            output.println("Some artifact transforms are not cacheable.  This can have negative performance impacts.  See more documentation here: " + documentationRegistry.getDocumentationFor("artifact_transforms", "artifact_transforms_with_caching") + ".");
+            output.println("Some artifact transforms are not cacheable.  This can have negative performance impacts.  See more documentation here: " + documentationRegistry.getDocumentationFor("artifact_transforms", "caching-transforms") + ".");
```

Comment:
I think this needs to be `sec:caching-transforms`

Reply:
You're right, it needed the "sec:".